tests/test_pubsub.py

```diff
@@ -474,3 +475,31 @@ def t_run(rc):
             t.start()
     except Exception:
         print("Error: unable to start thread")
+
+
+class TestPubSubPubSubSubcommands(object):
```

Comment:
I do not really get the hang of the class name in realtion to what is being tested here O.o Either change to something that i can understand or add a docstring that describeres the purpose of this seperate class instead of having the tests closer to the regular pubsub tests.